src/alias.js

```diff
@@ -158,7 +158,11 @@ class Alias {
 				if (_.isFunction(busboyOptions.onFileSizeLimit)) {
 					busboyOptions.onFileSizeLimit.call(this.service, file, busboy);
 				}
-				file.destroy(new PayloadTooLarge({ fieldname, filename, encoding, mimetype }));
+				file.destroy();
+				busboy.emit("error", new PayloadTooLarge({ fieldname, filename, encoding, mimetype }));
```

Comment:
Could you explain the changes?

Reply:
without this pull, below example will crash application
https://stackblitz.com/edit/node-3az7uh?file=index.js